blockchain.go

```diff
@@ -165,7 +165,7 @@ func (conf config) GetServiceKey() ServiceKey {
 	return serviceKey
 }
 
-const defaultGenesisTokenSupply = "10000000000.0"
+const defaultGenesisTokenSupply = "1000000000.0"
```

Comment:
Missed this change in a previous PR. This is only used in tests. Without this change storage on the service account overflows. See https://github.com/onflow/flow-emulator/pull/65 for context.